src/frontend/Felles/Visittkort/Visittkort.tsx

```diff
@@ -167,46 +173,49 @@ const VisittkortComponent: FC<{ data: IPersonopplysninger; behandling?: Behandli
                         <EtikettFokus mini>Fullmakt</EtikettFokus>
                     </ElementWrapper>
                 )}
-                {behandling && (
-                    <ElementWrapper>
-                        <>
+                <TagsWrapper>
+                    {behandling && (
+                        <ElementWrapper>
+                            <>
+                                <TagsLitenSkjerm>
```

Comment:
Burde vi kanskje wrappe disse i en egen komponent sånn att man kun trenger `<TagsLitenEllerStorSkjerm>...</.`  eller noe slik?

Reply:
Siden det er to forskjellige verdier som skal skrives ut så kan det bli litt "pussig". Så lenge vi har to av de så tenker jeg det er greit

Reply:
Ah, ser det nå. var en liten `kort` med der i tekstmappingen! 👍  Hade vel iofs vært mulig likevel

Reply:
Hmm ja, det er et alternativ å bare kalle det `<Tags>` og sende inn liten eller stor som props, men usikker på hva som blir best her. Holder det som det er foreløpig.